docs/catalogue/cat_network-data-manager.md

```diff
@@ -47,123 +44,98 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
+| dataset \* | Dataset name | e.g. core |
 | name \* | Variable name | e.g. height\_1 |
-| isRepeatOf.table \* | Table that contains the variable that is repeated | Tables must be predefined in the _TargetTables_ sheet; e.g. core |
-| isRepeatOf.name \* | Name of the variable that is repeated | Variables must be predefined in the _TargetVariables_ sheet; e.g. height\_0 |
-| collectionEvent.resource | Refer to the network that contains the collection event | e.g. LifeCycle |
-| collectionEvent.name | Refer to the name of a collection event | The collectionEvent needs to be predefined in the *CollectionEvents* sheet; e.g. y1 or y2 |
-
-<sup>Table 5. Description of the columns that can be filled out for RepeatedTargetVariables. * = mandatory</sup>
-
-#### *CollectionEvents* sheet
-
-The *CollectionEvents* sheet is optional. The timing of data collection in events is defined in the *CollectionEvents*
-sheet. It can be used to describe time periods within which the data for variables are collected. The events are defined
-here and referred to from the sheets *TargetVariables* and/or *RepeatedTargetVariables*.
-
-| *Column name* | *Description* | *Remarks* |
-| --- | --- | --- |
-| resource \* | Name of the model that contains this collection event | e.g LifeCycle_CDM |
-| name \* | Name of the collection event | e.g. y9 |
-| description | Event description | e.g. Between 9 and 10 years |
-| ageGroups | The age groups that were sampled within this collection event | Find list to choose from in CatalogueOntologies (AgeGroups) |
-| subcohorts | Subcohorts or subpopulations that are targeted with this variable | Subcohorts need to be predefined in the _Subcohorts_ sheet |
-
-<sup>Table 6. Description of the columns that can be filled out for CollectionEvents. * = mandatory</sup>
-
-#### *Subcohorts* sheet
-
-The sheet *Subcohorts* is optional. Here you may describe populations that can be linked to collection events.
-
-| *Column name* | *Description* | *Remarks* |
-| --- | --- | --- |
-| resource | Name of the model that contains this subcohort | e.g LifeCycle_CDM |
-| name \* | Name of the subpopulation or subcohort | e.g. mothers or children |
-| description | Subpopulation description | |
-
-<sup>Table 7. Description of the columns that can be filled out for Subpopulations. * = mandatory</sup>
+| is repeat of.table \* | Dataset that contains the variable that is repeated | Datasets must be predefined in the _Datasets_ sheet; e.g. core |
+| is repeat of.name \* | Name of the variable that is repeated | Variables must be predefined in the _Variables_ sheet; e.g. height\_0 |
+| collection event.resource | Refer to the network that contains the collection event | e.g. LifeCycle |
+| collection event.name | Refer to the name of a collection event | The collection event needs to be predefined via online forms, as described above; e.g. y1 or y2 |
+| since version | Version of the data model when this variable was introduced | e.g. 1.0.0 or 2.1 |
```

Comment:
since and until not defined in TargetDictionary.xlsx (Repeated variables)

Reply:
check